modules/widgets/gravatar-profile.css

```diff
@@ -40,7 +40,7 @@ only screen and (-o-min-device-pixel-ratio: 3/2),
 only screen and (min--moz-device-pixel-ratio: 1.5),
 only screen and (min-device-pixel-ratio: 1.5) {
 		.widget-grofile .grofile-accounts-logo {
-			background-image: url('//0.gravatar.com/images/grav-share-sprite-2x.png');
+			background-image: url('https://0.gravatar.com/images/grav-share-sprite-2x.png');
```

Comment:
It might be best to use `https://secure.gravatar.com/images/grav-share-sprite-2x.png` here.